pom.xml

```diff
@@ -65,6 +65,8 @@
         <commons-fileupload.version>1.3.3</commons-fileupload.version>
         <commons-io.version>2.6</commons-io.version>
         <commons-lang.version>2.6</commons-lang.version>
+        <integration.mysql.db.image>quay.io/eclipse/che--centos--mysql-57-centos7:latest-e08ee4d43b7356607685b69bde6335e27cf20c020f345b6c6c59400183882764</integration.mysql.db.image>
```

Comment:
it should use the one from sre mirror

Reply:
are you sure it's the same image?

Reply:
on https://registry.centos.org/centos/postgresql-96-centos7/latest (description tab) and on https://hub.docker.com/r/centos/postgresql-96-centos7 it seems that we see the same Dockerfile/source repository

![image](https://user-images.githubusercontent.com/436777/99679744-9344c100-2a7c-11eb-8695-9f095297d0a5.png)

Reply:
@skabashnyuk we don't want to maintain those images, hence I would rely on the ones maintained by app-sre.

Reply:
See comments on https://github.com/eclipse/che/issues/18292#issuecomment-730412565

Reply:
I think we can do that for mysql easily. For PostgreSQL, I would prefer to wait for @tolusha 's commits in operator and later will reuse the same.